daras_ai_v2/utils.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+
+def get_relative_time(timestamp):
+    now = datetime.now(timestamp.tzinfo)
+    diff = now - timestamp
+
+    seconds = diff.total_seconds()
+    if seconds < 2:
```

Comment:
Afaik, the timedelta object will already contain seconds, minutes, hours, days and weeks so i would avoid doing that maths

https://docs.python.org/3/library/datetime.html#timedelta-objects

Reply:
@devxpy updated this!